addons/binding/org.openhab.binding.airvisualnode/README.md

```diff
@@ -73,9 +65,68 @@ airvisualnode:avnode:1a2b3c4 [ address="192.168.1.32", username="airvisual", pas
 Here is an example of items for the AirVisual Node:
 
 ```
-Number Livingroom_Temperature "Temperature [%.1f °C]" <temperature> ["TargetTemperature"] {channel="airvisualnode:avnode:1a2b3c4:measurements#temp_celsius"}
-Number Livingroom_CO2_Level "CO₂" <flow> {channel="airvisualnode:avnode:1a2b3c4:measurements#co2_ppm"}
-Number Livingroom_Aqi_Level "Air Quality Index" <flow> { channel="airvisualnode:avnode:1a2b3c4:measurements#aqi_us" }
-Number Livingroom_Aqi_Pm25  "PM2.5 Level" <line> { channel="airvisualnode:avnode:1a2b3c4:measurements#pm_25" }
-DateTime Livingroom_Aqi_Timestamp "AQI Timestamp [%1$tH:%1$tM]" <clock> { channel="airvisualnode:avnode:1a2b3c4:status#timestamp" }
+Number:Temperature Livingroom_Temperature "Temperature [%.1f %unit%]" <temperature> {channel="airvisualnode:avnode:1a2b3c4:temperature"}
+Number:Dimensionless Livingroom_Humidity "Humidity [%d %unit%]" <humidity> {channel="airvisualnode:avnode:1a2b3c4:humidity"}
+Number:Dimensionless Livingroom_CO2_Level "CO₂" <flow> {channel="airvisualnode:avnode:1a2b3c4:co2"}
+Number:Dimensionless Livingroom_Aqi_Level "Air Quality Index" <flow> { channel="airvisualnode:avnode:1a2b3c4:aqi" }
+Number:Density Livingroom_Pm25_Level  "PM2.5 Level" <line> { channel="airvisualnode:avnode:1a2b3c4:pm_25" }
+DateTime Livingroom_Aqi_Timestamp "AQI Timestamp [%1$tH:%1$tM]" <clock> { channel="airvisualnode:avnode:1a2b3c4:timestamp" }
```

Comment:
Where do you have the category "clock" from? I cannot find it in our documentation: https://www.eclipse.org/smarthome/documentation/concepts/categories.html